### ISSUE TITLE
Docs article - registering top bar items

### ISSUE THREAD

**Issue by doitadrian** [webiny/docs.webiny.com#23]
## Prerequisites
1. Setup [`webiny-js`](https://github.com/webiny/webiny-js) repo - this is where Webiny is located. You're gonna want to read the [Contributing](https://github.com/webiny/webiny-js/blob/master/CONTRIBUTING.md) document.

2. Clone this repository - this is where you'll write the actual docs (the whole site is based on Docusaurus V2). A short intro can be found in the [`README`](https://github.com/webiny/docs.webiny.com/blob/master/README.md) file.
 
## Introduction

Once a new Webiny project is set up, out-of-the-box, users get access to the Admin app, which looks like this:
![image](https://user-images.githubusercontent.com/5121148/77308329-1699f100-6cfb-11ea-90ba-aba8f45a3af4.png)

The seen user interface is built using the concept of [plugins](https://docs.webiny.com/docs/deep-dive/plugins-crash-course), which is a very flexible way to add, remove, or alter existing functionality. Basically, a plugin is an object which consists primarily of `type` and `name` properties, and one or more custom properties that are needed to get the job done. For example:
```js
{ name: "my-plugin-xyz", "type": "my-plugin", ...otherXYZProps }
```

A good plugin usage example is the top bar in the Admin app, which utilizes three different types of plugins:

![image](https://user-images.githubusercontent.com/5121148/77308848-e4d55a00-6cfb-11ea-80dc-2d432638c842.png)

##### Who registered these plugins?
The answer is the `@webiny/app-admin` package (located in `packages/app-admin`), some examples can be found here:

- Global search in the middle - `packages/app-admin/src/plugins/GlobalSearch/index.tsx`
- Logo on the left - `packages/app-admin/src/plugins/Logo/index.tsx`
- User menu on the right: `packages/app-admin/src/plugins/UserMenu/index.tsx`

All plugins that the `@webiny/app-admin` package exports can be found here: `packages/app-admin/src/plugins/index.ts`

##### Where are these plugins imported?
You might be asking yourself, OK, I see this bunch of plugins, but how are they used? In the case of the Admin app (there is the Site app as well, but never mind that for now), all of the needed plugins are imported here `apps/admin/src/plugins/index.js`.

## The task

Your task is to play around with Webiny, and try registering additional components in the top bar. Once you've managed to do that, we want you to document it, so other users who will be using Webiny, know how to do it.

We already have [Adding routes](https://docs.webiny.com/docs/webiny-apps/routes-registration/adding-routes) and [Adding menus](https://docs.webiny.com/docs/webiny-apps/menus-registration/adding-menus) articles, so you can put your article here as well: 
![image](https://user-images.githubusercontent.com/5121148/77319383-07bd3980-6d0f-11ea-84dc-59e9f6fc5e75.png)

We can call it "Header Elements".

Please check the linked articles to get a feeling of how we approach the docs and what level of detail we like to apply.

## Plugin reference

Make sure to also add these plugins and short "more-formal" details in the [Plugins Reference > App](http://localhost:3001/docs/webiny-apps/admin/development/plugins-reference/app) page.

I've added some boilerplate text already for you in the `docs/webiny-apps/admin/development/plugins-reference/app.md`. It's all commented out, so remove the comments and just fill in the blanks.

![image](https://user-images.githubusercontent.com/5121148/77400709-e3fc0100-6dab-11ea-8658-4fc3a01ffd96.png)

**Comment by doitadrian** [comment]
I've added an additional thing for you to do - list and explain the plugins in the Plugins Reference section. Check out the added "Plugins reference" section in the issue.

**Comment by doitadrian** [comment]
Closed by #24
